xbmc/settings/AdvancedSettings.cpp

```diff
@@ -232,8 +232,8 @@ void CAdvancedSettings::Initialize()
   //m_videoStackRegExps.push_back("(.*?)([ ._-]*[0-9])(.*?)(\\.[^.]+)$");
 
   m_tvshowEnumRegExps.clear();
-  // foo.s01.e01, foo.s01_e01, S01E02 foo, S01 - E02
-  m_tvshowEnumRegExps.push_back(TVShowRegexp(false,"s([0-9]+)[ ._-]*e([0-9]+(?:(?:[a-i]|\\.[1-9])(?![0-9]))?)([^\\\\/]*)$"));
+  // foo.s01.e01, foo.s01_e01, S01E02 foo, S01 - E02, S01xE02
+  m_tvshowEnumRegExps.push_back(TVShowRegexp(false,"s([0-9]+)[ ._-x]*e([0-9]+(?:(?:[a-i]|\\.[1-9])(?![0-9]))?)([^\\\\/]*)$"));
```

Comment:
No when it's between `[` and `]`. In there nothing needs escaping.

What I'm not 100% sure about is the position of `-` because it has a special meaning between `[` and `]` like `[a-z]` so it might be safer to have that at the very beginning or very end to not be misinterpreted.